.github/workflows/test.yaml

```diff
@@ -105,28 +106,27 @@ jobs:
     runs-on: ubuntu-latest
     needs: test-matrix
 
-    container:
-      image: arduino/arduino-cli:builder-1
-      volumes:
-        # cache go dependencies across pipeline's steps
-        - ${{ github.workspace }}/go:/go
-
     steps:
       - name: checkout
         uses: actions/checkout@v1
         with:
           fetch-depth: 0
 
-      - name: build
-        shell: bash
+      - name: Install Taskfile
+        uses: arduino/actions/setup-taskfile@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          version: 3.x
+
+      - name: Build
         run: |
-          PACKAGE_NAME_PREFIX="${{ github.workflow }}"
+          PACKAGE_NAME_PREFIX="${{ github.workflow }}-"
           if [ "${{ github.event_name }}" = "pull_request" ]; then
-            PACKAGE_NAME_PREFIX="$PACKAGE_NAME_PREFIX-${{ github.event.number }}"
+            PACKAGE_NAME_PREFIX="$PACKAGE_NAME_PREFIX-${{ github.event.number }}-"
```

Comment:
The `-` appended here are redundant. Note the double `--` here:
https://github.com/arduino/arduino-cli/pull/1148/checks?check_run_id=1748258721#step:4:91
```
--build-cmd "go build -o dist/arduino-cli_windows_386/arduino-cli.exe -ldflags ' -X github.com/arduino/arduino-cli/version.versionString=test--1148--6a52a9a31b2b71bf80f20fa03205143adf07d4cf-git-snapshot -X github.com/arduino/arduino-cli/version.commit=e975be30 -X github.com/arduino/arduino-cli/version.date=2021-01-22T11:14:28Z '
```
```
$ ./arduino-cli version
arduino-cli.exe alpha Version: test--1148--6a52a9a31b2b71bf80f20fa03205143adf07d4cf-git-snapshot Commit: e975be30 Date: 2021-01-22T11:14:28Z
```

Reply:
Right, I'll remove it.